trin-core/src/jsonrpc/utils.rs

```diff
@@ -36,6 +36,17 @@ pub fn bucket_entries_to_json(bucket_entries: BTreeMap<usize, Vec<NodeTuple>>) -
                         map.insert("enr".to_owned(), enr.to_base64());
                         map.insert("status".to_owned(), format!("{:?}", node_status.state));
                         map.insert("radius".to_owned(), format!("{}", data_radius));
```

Comment:
Could you also add the `ip` address in here (`enr.ip4()` ... I think)? I've been doing this locally and it helps identify the client you're talking to.... Actually, how about you only output the ip if there is not client info found, to reduce clutter?

Reply:
Good idea, done.

I added a note that it can be removed once we have a nice dashboard, because then we can move from reading raw RPC output (like barbarians) and just decode the ENR UI-client-side.

Reply:
@njgheorghita weird, with that change that you proposed I seem to get a failing utp test and failing peertest. That doesn't sound right, going to investigate further.

Reply:
> @njgheorghita weird, with that change that you proposed I seem to get a failing utp test and failing peertest in CI. That doesn't sound right, going to investigate further.

I'm curious, what was the resolution to this?